test/integration/patients/patient/sidebar.js

```diff
@@ -21,12 +22,14 @@ context('patient sidebar', function() {
 
         return fx;
       })
+      .routePatientEngagementStatus('active')
```

Comment:
what would happen if this was the only use of `routePatientEngagementStatus` in the tests?

Reply:
Oh good call. Now that we're handling failed calls correctly we don't need it everywhere there is a patient call.